lib/src/main/java/com/diffplug/spotless/FormatExceptionPolicyLegacy.java

```diff
@@ -15,13 +15,13 @@
  */
 package com.diffplug.spotless;
 
-import java.util.logging.Level;
-import java.util.logging.Logger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 class FormatExceptionPolicyLegacy extends NoLambda.EqualityBasedOnSerialization implements FormatExceptionPolicy {
 	private static final long serialVersionUID = 1L;
 
-	private static final Logger logger = Logger.getLogger(Formatter.class.getName());
+	private static final Logger logger = LoggerFactory.getLogger(Formatter.class.getName());
```

Comment:
:question: IntelliJ/SonarLint warns that we're using the wrong class here - should we amend this to:
```suggestion
	private static final Logger logger = LoggerFactory.getLogger(FormatExceptionPolicyLegacy.class.getName());
```